### DISCUSSION TITLE
i mistakenly delted my organization what to do now?

### DISCUSSION THREAD

**Discussion by engi-harsh** [community/community#142871] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I accidentaly delted my GIthub organization with username "harshuopjs"
please help to retrieve it as it has a lot of crucial data i will be finished ssly help me please as soon as possible

**Comment by peachmontt** [comment]
I'm sorry to hear about the accidental deletion of your GitHub organization. Unfortunately, GitHub doesn't provide a direct way to recover a deleted organization, but you can try the following steps:

1. Contact GitHub Support
Visit GitHub Support: Go to the [GitHub Support page](https://support.github.com/contact).
Select the Appropriate Category: Choose "Account" or "Organization" as your issue type.
Provide Details: Clearly explain your situation, including your organization name (harshuopjs), when it was deleted, and any relevant information about the repositories or data you had.
2. Check Your Email
If your organization was deleted recently, check your email for any notifications from GitHub. There may be instructions or options provided in those emails.
3. Verify Your Backups
If you have backups of your repositories (like local clones), you can recover some of your work, but the organization itself (permissions, settings, etc.) may not be recoverable.
4. Follow Up
After submitting a request to GitHub Support, monitor your email for a response. They may ask for additional information to verify your identity or provide further assistance.
Note
Recovering a deleted organization is not guaranteed, but GitHub Support is your best bet in this situation. They may have specific processes for handling such requests.

I hope you can recover your organization soon! If you have any other questions or need further assistance, feel free to ask.
